lib/rules/disallow-space-between-arguments.js

```diff
@@ -46,20 +46,13 @@ module.exports.prototype = {
     },
 
     check: function(file, errors) {
-
-        var tokens = file.getTokens();
-
         file.iterateNodesByType(['CallExpression'], function(node) {
-
             node.arguments.forEach(function(param, i) {
-
-                var punctuatorToken = file.getTokenByRangeStart(param.range[1]);
-
+                var punctuatorToken = file.getPrevToken(file.getFirstNodeToken(param));
```

Comment:
The `punctuatorToken.value === ','` is important
Ok I should just rename to this then.

```js
var nextToken = file.getFirstNodeToken(param);
                var punctuatorToken = file.getPrevToken(nextToken);
                if (punctuatorToken.value === ',') {
                    errors.assert.noWhitespaceBetween({
                        token: punctuatorToken,
                        nextToken: file.getNextToken(punctuatorToken)
                    });
                }
```